bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestListTables.java

```diff
@@ -85,8 +88,34 @@ public void testTableNames() throws IOException {
         Assert.assertTrue(ArrayUtils.contains(tableList, tableName2));
       }
 
-      checkColumnFamilies(admin, tableName2);
+      {
+        TableName[] tableList = admin.listTableNames(Pattern.compile("list_table1-.*"));
+        Assert.assertTrue(ArrayUtils.contains(tableList, tableName1));
+        Assert.assertFalse(ArrayUtils.contains(tableList, tableName2));
```

Comment:
if we are expecting only one element in the array.
we should do:
1. check size == 1
2. [0] == expect

or 
some util that asserts array has exactly has one element == expected